hs_core/tasks.py

```diff
@@ -844,6 +844,7 @@ def create_temp_zip(resource_id, input_path, output_path, aggregation_name=None,
         istorage.zipup(output_path, *set(files_to_zip), in_prefix=os.path.dirname(input_path))
     else:  # regular folder to zip
         istorage.zipup(output_path, input_path)
+    res.update_download_count()
```

Comment:
I'm trying to see if this would create miscounts in these cases:

1. https://github.com/hydroshare/hydroshare/blob/rc-3.11.3/hs_core/views/__init__.py#L873
2. https://github.com/hydroshare/hydroshare/blob/rc-3.11.3/hs_core/views/utils.py#L1115